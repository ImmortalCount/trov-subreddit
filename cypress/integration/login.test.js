// describe('Login page', () => {
//     it('should render the login page', () => {
//         cy.visit('/').then(() => {
//             cy.get('.login-form').toMatchSnapshot()
//         })
//     })

//     it('should display errors', () => {
//         cy.visit('/')
//         cy.get('input').first().type('jason.klingstedt@gmail.com')
//         cy.get('input').last().type('password')
//         cy.get('form').submit()
//     })

//     it('should log on as admin', () => {
//         cy.login('admin')
//         cy.visit('/').then(() => {
//             cy.get('#root').toMatchSnapshot()
//         })
//     })
// })
