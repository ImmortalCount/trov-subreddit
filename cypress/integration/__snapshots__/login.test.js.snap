exports[`Login page > should render the login page #0`] = `
<div class="ui message login-form">
  <form novalidate="">
    <div class="ui form">
      <div class="field">
        <label for="username"> Email Address </label>
        <div class="ui massive input">
          <input name="username" id="username" type="text" value="" />
        </div>
      </div>
      <div class="field">
        <label for="password"> Password </label>
        <div class="ui massive input">
          <input name="password" id="password" type="password" value="" />
        </div>
      </div>
      <button class="ui green button">Submit</button>
    </div>
  </form>
</div>
`;

exports[`Login page > should log on as admin #0`] = `
<div id="root">
  <div class="main-container">
    <nav>
      <div class="ui pointing menu">
        <div class="item">
          <a class="active" aria-current="page" href="/">Home</a>
        </div>
        <div class="right menu">
          <div class="item user-menu">
            <p class="user-name">Jason Klingstedt</p>
            <span class="user-sign-out"
              >Sign Out<i
                aria-hidden="true"
                class="log out icon sign-out-icon"
              ></i
            ></span>
          </div>
        </div>
      </div>
    </nav>
    <div>Home</div>
  </div>
</div>
`;
